File: src/main/java/de/aquafun3d/bingo/utils/inventories/ITeamInventories.java
package de.aquafun3d.bingo.utils.inventories;

import org.bukkit.entity.Player;
import org.bukkit.inventory.Inventory;
import org.bukkit.inventory.ItemStack;
import org.bukkit.scoreboard.Team;

public interface ITeamInventories {

    void fillInventories();
    void removeItem(Player player, ItemStack item);
    Inventory getIventorybyPlayer(Player player);
    Inventory getInventorybyTeam(Team team);
    ItemStack getItem();
}

File: src/main/java/de/aquafun3d/bingo/utils/inventories/ITeamInventories.kt
package de.aquafun3d.bingo.utils.inventories

import org.bukkit.Material
import org.bukkit.entity.Player
import org.bukkit.inventory.Inventory
import org.bukkit.inventory.ItemStack
import org.bukkit.scoreboard.Team

interface ITeamInventories {
    fun fillInventories()
    fun removeItem(player: Player, item: ItemStack)
    fun removeForAll(player: Player, item: Material)
    fun removeItem(player: Player, item: Material)
    fun removeItemIndex(index: Int, player: Player)
    fun getInventorybyPlayer(player: Player): Inventory
    fun getInventorybyTeam(team: Team): Inventory
    fun getItem(): ItemStack
    fun itemCount(player: Player): Int
}
